Scripts/build.swift

```diff
@@ -174,6 +174,9 @@ for rawPlatform in rawPlatforms {
     if platform.shouldTest {
         xcodeBuildArguments.append("test")
     }
+    xcodeBuildArguments.append("-test-iterations")
+    xcodeBuildArguments.append("100")
+    xcodeBuildArguments.append("-run-tests-until-failure")
```

Comment:
This flag stops running tests once they fail. It respects the iterations limit above.